src/AzureStack/Admin/BackupAdmin/Backup.Admin/Microsoft.AzureStack.Management.Backup.Admin.csproj

```diff
@@ -16,6 +16,9 @@
                 1) Configure where backups are stored
                 2) Perform backups 
                 3) List and restore completed backup. 
+                
+                0.2.0-preview breaking change:
```

Comment:
Replace the first three points with this text

Reply:
Sure. Thanks.